mon-pix/app/metrics-adapters/pix-matomo-tag-manager-adapter.js

```diff
@@ -0,0 +1,47 @@
+import { assert } from '@ember/debug';
+import BaseAdapter from 'ember-metrics/metrics-adapters/base';
+
+export default class PixMatomoTagManager extends BaseAdapter {
+  toStringExtension() {
+    return 'PixMatomoTagManager';
```

Comment:
Si l'idee est d'aligner sur le nom de la classe, je propose ceci pour ne pas avoir a maintenir la correlation

```suggestion
    return this.constructor.name;
```